utils/utils.go

```diff
@@ -83,3 +83,23 @@ func GetBoolFromEnvWithDefault(enVarName string, defaultValue bool) bool {
 
 	return envBool
 }
+
+// PanicIfErr panics if err != nil
+func PanicIfErr(err error) {
```

Comment:
FYI:
I would find `MustBeNil` a bit easier to read

Reply:
Or `PanicIfNotNil` - that would also describe what the comment does

Reply:
I actually think `PanicIfErr` is quite descriptive, but `PanicIfNotNil` is also fine.
`MustBeNil` doesn't really tell what it does so I would prefer one of the other two.

Reply:
The standard library often uses `Must` to indicate something that will panic (like `regexp.MustCompile` or `template.Must` ) and I've seen a few third-party libraries doing the same, so this is kind of a convention I think.
But this should only be used for things that we expect to never fail (like Marshalizing or Unmarshalizing a piece of json that we have written ourselves).
For things that require a graceful shutdown, we should add a primitive to the executor

Reply:
For now, I would be fine with having both `MustBeNil` and `PanicIfNotNil`:
- use `MustBeNil` for things that we *really* don't expect to happen
- use `PanicIfNotNil` for things that we will replace with a proper graceful shutdown later

Reply:
I implemented both functions now but only changed `PanicIfErr` to `PanicIfNotNil` now because its hard to determine the difference in each case